academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md

```diff
@@ -96,6 +96,111 @@ The sku for MAKES API host machines.
 
 Whether to clean up resources if deployment failed for any reason. Used for debugging MAKES host deployment failures.
 
+## CreateIndexResources Command
+
+Creates the MAKES index building resources. E.g. MAKES index build batch account, storage account.
+
+```cmd
+kesm CreateIndexResources --IndexResourceName
+                        --MakesPackage
+                        --MakesIndexResourceConfigFilePath
+                        --[Region]
+                        --[MakesPreprocessor]
+                        --[MakesIndexer]
+                        --[MakesJobManager]
+```
+
+### Required Parameters
+
+`--IndexResourceName`
+
+The name for the indexing resources. Indexing resources name should be less than 64 characters with only numbers and lower case letters.
+
+`--MakesPackage`
+
+The base URL to a MAKES release package.
+
+`--MakesIndexResourceConfigFilePath`
+
+Outputs the MAKES indexing resources config file for BuildIndex command. The command won't write a config file unless a path is provided.
+
+### Optional Parameters
+
+`--Region`
+
+The region to create the indexing resources in.
+
+`--MakesPreprocessor`
+
+The MAKES preprocessor zip url.
+
+`--MakesIndexer`
+
+The MAKES indexer zip url.
+
+`--MakesJobManager`
+
+The MAKES JobManager zip url.
+
+## BuildIndex Command
+
+Builds MAKES index(es) from json entities.
+
+```cmd
+kesm BuildIndex --MakesIndexBuildResourceConfigFilePath
+                --EntitiesUrlPrefix
+                --OutputUrlPrefix
+                --[IndexPartitionCount]
+                --[IntersectionMinCount]
+                --[MakesPreprocessor]
+                --[MakesIndexer]
+                --[MakesJobManager]
+                --[MaxStringLength]
+                --[RemoveEmptyValues]
+                --[WorkerCount]
+                --[WorkerMachineSku]
+```
+
+### Required Parameters
+
+`--MakesIndexBuildResourceConfigFilePath`
+
+The MAKES indexing resource configuration file path. Run CreateIndexResource command to generate one.
+
+`--EntitiesUrlPrefix`
+
+The input entities file Url prefix.
+
+`--OutputUrlPrefix`
+
+The output url prefix for writing built index.
+
+### Optional Parameters
+
+`--IndexPartitionCount`
+
+The number of index partitions to create.
```

Comment:
Should we provide guidance somewhere as to the optimal number to create?  Is that in later documentation?